packages/fluentui/react-northstar/test/specs/components/Datepicker/navigateToNewDate-test.tsx

```diff
@@ -0,0 +1,32 @@
+import { navigateToNewDate } from 'src/components/Datepicker/navigateToNewDate';
+import { addDays, addMonths, addWeeks, compareDatePart } from '@fluentui/date-time-utilities';
+
+describe('navigateToNewDate', () => {
+  const referenceDate = new Date(2020, 0, 10);
+
+  it('Navigation works for day', () => {
+    const result = navigateToNewDate(referenceDate, 'Day', 1, {});
+    expect(compareDatePart(result, addDays(referenceDate, 1))).toBeFalsy();
```

Comment:
Let's use dates here instead of calling other function, that way we can be sure it won't regress in the future.